torch/onnx/_internal/fx/serialization.py

```diff
@@ -111,6 +112,7 @@ def save_model_with_external_data(
         onnx_model: ONNX model to be saved with external initializers.
             If an input name matches a tensor loaded from "torch_load_paths",
             the tensor will be saved as that input's external initializer.
+        rename_initializer: Replaces "." by "_" for all ONNX initializer names.
```

Comment:
Let's add some explanation. We want to call out this is only needed by fx_symbolic_tracer path.

Reply:
There was one at https://github.com/pytorch/pytorch/pull/105002/files#diff-0c7a4333620a22a5c3e5315e30272b59fb7a11b393cb42f8255070bedeb02738L125, but I will add in the docstring too (in fact, I had added before arghh)